src/routes/(tools)/invoice-generator/+page.svelte

```diff
@@ -0,0 +1,1917 @@
+											<th class="text-end w-1/6 px-3">Quantity</th>
+											<th class="text-center w-fit">Amount</th>
+											<th class="text-center w-fit">Tax(%)</th>
+										</tr>
+										<tr>
+											<td class="border-t border-black" colspan="6">
+												<hr />
+											</td>
+										</tr>
+									</thead>
+									<tbody>
+										{#each inputSets as inputSet}
+											{#if showFirstPair || index > 0}
+												<tr>
+													<td>
+														<button
+															class="w-10 h-10 font-bold border text-xl bg-gray-400 rounded-lg m-3"
+															on:click={() => deleteInputSet(inputSet.id)}>X</button
+														>
+													</td>
+													<td>
+														<input
+															class="w-full rounded"
+															type="text"
+															min="0"
+															placeholder="Item Description"
+															bind:value={inputSet.item}
+														/>
+													</td>
+													<td class="px-2">
+														<input
+															class="w-full text-end rounded"
+															type="number"
+															min="0"
+															placeholder="0.00"
+															bind:value={inputSet.rate}
+															on:change={updateTotals}
+														/>
+													</td>
+													<td class="px-1">
+														<input
+															class="w-full rounded text-end"
+															type="number"
+															min="0"
+															bind:value={inputSet.quantity}
+															on:change={updateTotals}
+														/>
+													</td>
+													<td>
+														<p class="flex justify-center text-2xl">
+															₹{inputSet.quantity * inputSet.rate}
+														</p>
+													</td>
+													<td class="text-center">
+														<input
+															class="w-16 text-center text-l rounded"
+															type="number"
+															min="0"
+															bind:value={inputSet.taxRate}
+															on:change={updateTotals}
+														/>
+													</td>
+												</tr>
+												<tr>
+													<td colspan="6">
+														<textarea
+															class="w-2/5 ml-16 rounded h-36 text-start pt-2 resize-none"
+															placeholder="Additional Details"
+															bind:value={inputSet.additionalInfo}
+														/>
+													</td>
+												</tr>
+												{#if index !== inputSets.length - 1}
+													<tr>
+														<td class="py-3" colspan="6">
+															<hr />
+														</td>
+													</tr>
+												{/if}
+											{/if}
+										{/each}
+									</tbody>
+								</table>
+							</div>
+							<button
+								class="w-10 h-10 border font-bold text-3xl text-white bg-gray-600 rounded-lg shadow-lg m-3"
+								on:click={addInputSet}>+</button
+							>
+							<hr class="py-5" />
+							<div
+								class="gap-12 items-center mx-auto max-w-screen-xl overflow-hidden rounded-lg lg:grid lg:grid-cols-2"
+							><script lang="ts">
```

Comment:
Seems there have been merging issue. Multiple script and DOM elements are combined together,